var/spack/repos/builtin/packages/python/package.py

```diff
@@ -805,8 +808,9 @@ def import_tests(self):
                 self.command("-c", "import xml.parsers.expat")
                 self.command("-c", "import xml.etree.ElementTree")
 
-            # Ensure that ctypes module works
-            if "+ctypes" in spec:
+            # Ensure that ctypes module works on
+            # linux/darwin unconditionally and on windows conditionally
+            if sys.platform != win32 or (sys.platform == win32 and "+ctypes" in spec):
```

Comment:
The `ctypes` spec is `when="platform=windows"`, so the check for Windows is redundant here, `+ctypes` is only a Windows option. 
```suggestion
            if sys.platform != win32 or spec.satisfies("+ctypes"):
```